practice-two/src/components/FormInput/FormInput.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
```

Comment:
what do you mean form input?

Reply:
I named it like that because it's a input filed of a form validation that I will you to make a form to create and update customer.

Reply:
but the component just render input and error not validate.
Please make the naming is right